test/JDBC/expected/Test-Identity-before-14_7-or-15_2-vu-verify.out

```diff
@@ -0,0 +1,314 @@
+EXEC test_identity_vu_prepare_p1
+GO
+~~ROW COUNT: 1~~
+
+~~START~~
+int
+2
+~~END~~
+
+~~START~~
+numeric
+2
+~~END~~
+
+~~START~~
+numeric
+2
+~~END~~
+
+~~START~~
+numeric
+2
+~~END~~
+
+
+-- SCOPE_IDENTITY should return NULL because insert into t1 happened inside a function
+-- Validated the same behaviour in SQLServer
+SELECT MAX(id) as MaximumUsedIdentity FROM test_identity_vu_prepare_t1
+SELECT SCOPE_IDENTITY()
+SELECT @@IDENTITY
+SELECT IDENT_CURRENT('test_identity_vu_prepare_t1')
+GO
+~~START~~
+int
+2
+~~END~~
+
+~~START~~
+numeric
+<NULL>
+~~END~~
+
+~~START~~
+numeric
+2
+~~END~~
+
+~~START~~
+numeric
+2
+~~END~~
+
+
+EXEC test_identity_vu_prepare_p1
+GO
+~~ROW COUNT: 1~~
+
+~~START~~
+int
+3
+~~END~~
+
+~~START~~
+numeric
+3
+~~END~~
+
+~~START~~
+numeric
+3
+~~END~~
+
+~~START~~
+numeric
+3
+~~END~~
+
+
+SELECT * FROM test_identity_vu_prepare_t1
+GO
+~~START~~
+int#!#nvarchar
+1#!#Nirmit_Shah
+2#!#Nirmit_Shah
+3#!#Nirmit_Shah
+~~END~~
+
+
+-- SCOPE_IDENTITY should not be the same as IDENTITY
+-- SCOPE_IDENTITY matches max(id) in t2
+-- IDENTITY is the value from trigger
+-- validated same behaviour in SQLServer
+EXEC test_identity_vu_prepare_p2
+GO
+~~ROW COUNT: 1~~
+
+~~ROW COUNT: 1~~
+
+~~START~~
+smallint
+5
+~~END~~
+
+~~START~~
+numeric
+5
+~~END~~
+
+~~START~~
+numeric
+120
+~~END~~
+
+~~START~~
+numeric
+5
+~~END~~
+
+
+-- insert into t3, no triggers during insert into t3
+-- all identities should be the same
+-- validated in SQL Server
+EXEC test_identity_vu_prepare_p3
+GO
+~~ROW COUNT: 1~~
+
+~~START~~
+int
+125
+~~END~~
+
+~~START~~
+numeric
+125
+~~END~~
+
+~~START~~
+numeric
+125
+~~END~~
+
+~~START~~
+numeric
+125
+~~END~~
+
+
+SELECT * FROM test_identity_vu_prepare_t2
```

Comment:
I think we need order by here

Reply:
The query is the following I dont think we need order by

```
SELECT MAX(id) as MaximumUsedIdentity FROM test_identity_vu_prepare_t1
SELECT SCOPE_IDENTITY()
SELECT @@IDENTITY
SELECT IDENT_CURRENT('test_identity_vu_prepare_t1')
GO
```

Reply:
done